### PR TITLE
[Test] gamm: Make test case for GetPoolDenoms

### PR DESCRIPTION
**Pull request by stackman27**
Closes: #2200

## What is the purpose of the change
Add test for function GetPoolDenoms 

## Brief Changelog
n/a

## Testing and Verifying
added unit test 

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

File: x/gamm/keeper/pool.go [hunk 1]
```diff
@@ -198,8 +198,14 @@ func (k Keeper) DeletePool(ctx sdk.Context, poolId uint64) error {
 // 	return nil
 // }
 
+// GetPoolDenom retrieves the pool based on PoolId and
+// returns the coin denoms that it holds.
 func (k Keeper) GetPoolDenoms(ctx sdk.Context, poolId uint64) ([]string, error) {
 	pool, err := k.GetPoolAndPoke(ctx, poolId)
+	if err != nil {
+		return nil, err
```
Comment (p0mvn) [inline x/gamm/keeper/pool.go:206]: let's please also add a case where poolId does not exist so that we cover this branch

Comment (stackman27) [reply x/gamm/keeper/pool.go:206]: added here: [`c59f147`](https://github.com/osmosis-labs/osmosis/pull/2240/commits/c59f147bb825ef80b00dda548d7261e3a39cb2be)

File: x/gamm/keeper/pool_service_test.go [hunk 2]
```diff
@@ -751,6 +751,45 @@ func (suite *KeeperTestSuite) TestJoinSwapExactAmountInConsistency() {
...
+			expectErr:    false,
+		},
+		{
+			desc:         "Invalid PoolId",
+			poolId:       2,
+			expectDenoms: []string{"bar", "foo"},
+			expectErr:    true,
+		},
+	} {
+		suite.Run(fmt.Sprintf("Case %s", tc.desc), func() {
```
Comment (p0mvn) [inline x/gamm/keeper/pool_service_test.go:780]: Might be a personal preference here so feel free to ignore if you disagree - I see little value in the Case prefixes and I feel like they are only making it harder to identify distinct cases printed in the command line, especially when there are many cases
```suggestion
		suite.Run(tc.desc, func() {
```

**Review by p0mvn — APPROVED** [review]
Nice work, LGTM

File: x/gamm/keeper/pool_service_test.go (hunk 2 above)
Comment (p0mvn) [reply x/gamm/keeper/pool_service_test.go:780]: Please ping me once you decide whether to address it, and I will merge this PR :+1:

Comment (stackman27) [reply x/gamm/keeper/pool_service_test.go:780]: oh yea i can add that. I dont have a strong opinion on it

**Pull request merged**